web/src/features/comments/validateCommentReferenceObject.ts

```diff
@@ -39,27 +48,42 @@ export const validateCommentReferenceObject = async ({
   input: z.infer<typeof CreateCommentData>;
 }): Promise<{ errorMessage?: string }> => {
   const { objectId, objectType, projectId } = input;
-  const prismaModel = COMMENT_OBJECT_TYPE_TO_PRISMA_MODEL[objectType];
 
-  if (!prismaModel) {
-    return {
-      errorMessage: `No prisma model for object type ${objectType}`,
-    };
-  }
+  if (env.CLICKHOUSE_URL && isObservationOrTrace(objectType)) {
```

Comment:
Is this v3 behavior only? If yes, you can kill the env check.

Reply:
Yes, will not cherry pick for v2. Great call :)